src/cli/commands/workspaces.js

```diff
@@ -0,0 +1,36 @@
+// @flow
+
+import type Config from '../../config.js';
+import {MessageError} from '../../errors.js';
+import type {Reporter} from '../../reporters/index.js';
+
+const invariant = require('invariant');
+
+export function setFlags(commander: Object) {}
+
+export function hasWrapper(commander: Object, args: Array<string>): boolean {
+  return true;
+}
+
+export async function run(config: Config, reporter: Reporter, flags: Object, args: Array<string>): Promise<void> {
+  const {workspaceRootFolder} = config;
+
+  if (!workspaceRootFolder) {
+    throw new MessageError(reporter.lang('workspaceRootNotFound', config.cwd));
+  }
+
+  const manifest = await config.findManifest(workspaceRootFolder, false);
+  invariant(manifest && manifest.workspaces, 'We must find a manifest with a "workspaces" property');
```

Comment:
_We need a manifest at a higher level with `workspaces` defined in it._

Reply:
The wording doesn't matter much here since it's an invariant